optaplanner-core/src/main/java/org/optaplanner/core/api/score/holder/ScoreHolder.java

```diff
@@ -81,7 +87,9 @@
      * @return never null
      * @throws IllegalStateException if {@link #isConstraintMatchEnabled()} is false
      * @see ScoreDirector#getConstraintMatchTotals()
+     * @deprecated for removal from public API
```

Comment:
Let's reference ScoreManager because if they're using this, they probably want to learn about ScoreManager.

Reply:
Even though those getConstraintMatchTotals() methods aren't there yet, they will end up on ScoreManager. Might as well already reference it.